typescript/sdk/src/deploy/core/HyperlaneCoreChecker.ts

```diff
@@ -15,6 +17,15 @@ import {
   ThresholdViolation,
 } from './types';
 
+const MAILBOX_WITHOUT_LOCAL_DOMAIN_BYTE_CODE_HASH =
```

Comment:
How were these generated?

Reply:
I ran the checker and took the `actual` values from the violations